tests/test.rs

```diff
@@ -1,11 +1,14 @@
 #[macro_use]
 extern crate structure;
 
-use std::os::raw::c_void;
-use std::mem::transmute;
-use std::io::ErrorKind;
-use std::io::Cursor;
+#[allow(unused_imports)]
```

Comment:
It looks like only `ErrorKind` and `Cursor` are needed, remove the rest of the imports?

Reply:
Fixed in 6da4216. Will squash these commits into one, before the PR is merged.